sdk/storage/azure-storage-blob-nio/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## 12.0.0-beta.17 (Unreleased)
 
 ### Features Added
+- Added `AzurePath.fromBlobUrl` to help convert from a blob url to an AzurePath
+- Added a configuration option `AZURE_STORAGE_SKIP_INITIAL_CONTAINER_CHECK` to skip the initial container check in cases where the authentication method used will not have necessary permissions.
```

Comment:
there doesn't seem to be a reference to this option in the PR.

Reply:
Mishandled a merge with main here and this is a duplicate entry. Removing